vcd/resource_vcd_external_network_v2.go

```diff
@@ -142,9 +142,21 @@ func resourceVcdExternalNetworkV2() *schema.Resource {
 				Optional:    true,
 				Description: "Network description",
 			},
+			"dedicated_org_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "Dedicate External Network to an Org (only with IP Spaces)",
+			},
+			"use_ip_spaces": {
```

Comment:
This field name relates to my question above. We should consider maybe it's worth using `use_ip_spaces` (compared to `uses_ip_spaces`) across the board. Just opening up for thoughts.

Reply:
No idea if it can, but I think we can rename it without big doubt. It still remains `Computed` for now. The decision, if Edge Gateway is backed by IP Spaces for now, is decided by the backing Provider Gateway (`vcd_external_network_v2`) and if it has IP Space Uplinks (`vcd_ip_space_uplink`)

Reply:
Changed to `uses_ip_spaces` here, resource, tests and in website docs. Also ran a test to see that it still passes

Reply:
(Trying to post a reply second time - GitHub is misbehaving).

I think you have meant `use_ip_spaces` and it looks great!